### ISSUE TITLE
paperwork-shell missing after pypi-install installation

### ISSUE THREAD

**Issue by nodiscc** [jflesch/paperwork#575]
After following the Debian installation instructions (using `pypi-install`, https://github.com/jflesch/paperwork/blob/unstable/doc/install.debian.markdown#install-through-packages-generated-by-python-stdeb), there is no paperwork-shell command available.

```
$ sudo pypi-install pyocr; sudo pypi-install pyinsane; sudo pypi-install paperwork
...
$ paperwork-shell chkdeps paperwork_backend
bash: paperwork-shell: command not found
```

Edit: this file is properly installed when installing through virtualenv/pip: `python3 -m venv ~/tmp/venv; source ~/tmp/venv/bin/activate; pip install paperwork; paperwork-shell chkdeps paperwork_backend` and works as expected. Only the `pypi-install` method seems to have this problem.

**Comment by tiramiseb** [comment]
It seems the instructions on the aforementioned page are missing the following command:
```
$ sudo pypi-install paperwork-backend
```

@jflesch will be able to confirm that...

**Comment by jflesch** [comment]
paperwork-backend should be installed by dependency :/

**Comment by jflesch** [comment]
Doc wasn't up-to-date, sorry. It's fixed: 6e8fdb7ad4106879e2c5fe2b43d981244b382a24

**Comment by nodiscc** [comment]
Thank you
